.github/workflows/docs_bot.yml

```
@@ -0,0 +1,17 @@
+
+name: docs-bot
+on:
+  status
+jobs:
+  run-docs-bot:
+    if: ${{ github.repository == 'apache/tvm' && github.event.state == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Comment link to docs
+        env:
+          PR: ${{ toJson(github.event.pull_request) }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -eux
+          python tests/scripts/github_docs_comment.py || echo step failed
```

this job doesn't show up on PRs afaik so we don't need to hide failures

```suggestion
          python tests/scripts/github_docs_comment.py
```